annotation/src/main/scala/org/virtuslab/ash/annotation/Serializer.scala

```diff
@@ -2,5 +2,19 @@ package org.virtuslab.ash.annotation
 
 import scala.annotation.nowarn
 
+/**
+ * This annotation is used by Codec Registration Checker for marking serializer.
+ *
+ * Compiler plugin takes the body of marked class, collects all types that are present, filters them with provided regex and extracts type parameters.
+ * Then filtered types and extracted type parameters are checked against all direct subtypes of `clazz`.
+ *
+ * Sometimes types appear unexpectedly during type class derivation leading to false negatives.
+ * The role of `typeRegexPattern` is to filter detected types, ensuring they are used in the right context.
+ *
+ * For example if we want to serialize `trait Command`, then type `Registration[Command]` is relevant, while `Option[Command]` is not.
```

Comment:
Aaand pls provide the corresponding `typeRegexPattern` for this case... also, would be nice to mention that our Circe serializer is referenced here (`if we want to serialize trait Command with circe-akka-serializer` ?)

Reply:
🤔 resolved?